### ISSUE TITLE
Selection not working on Samsung Galaxy Tab2

### ISSUE THREAD

**Issue by swatisachdeva** [btate/BTAndroidWebViewSelection#22]
Hi,
First of all I really appreciate the hard work as this is a very nice tool which we have been using in our app.
But the selection seems not to work on Samsung Tab since the default behaviour on the samsung devices is to show their own text selection tool which has copy search and other menu items and hence I am unable to use the custom text selection on the device. Any help or advice would be really appreciated as any workaround is not working too well for me.

**Comment by btate** [comment]
Are you using the latest version with the latest javascript, etc?  I have a galaxy tab 2 in front of me and it works fine.

**Comment by swatisachdeva** [comment]
Yes I am using the latest javascript and the latest version of code. But on samsung tab2, it is just not working.

**Comment by swatisachdeva** [comment]
The onLongClick method for a webview in a samsung device is never called, i dun understand how things are working at your end. Can you just check if the function onLongClick is called automatically after onTouch in Samsung Tab? That would be of a great help. Thanks a lot

**Comment by btate** [comment]
It gets called on my end.  What version of android are you running?  I'm on 4.1

**Comment by swatisachdeva** [comment]
I am using Android version 4.1.2 and I had to call the long click method explicitly if the touch is for more then few seconds but that effects the selection part on other devices like Nexus then, I am still struggling to find a better solution.
